modules/tensor_mechanics/src/materials/ComputeFiniteStrain.C

```diff
@@ -82,48 +82,51 @@ ComputeFiniteStrain::computeProperties()
     ave_Fhat /= _current_elem_volume;
     // average deformation gradient
     ave_dfgrd_det /= _current_elem_volume;
-  }
 
-  for (_qp = 0; _qp < _qrule->n_points(); ++_qp)
-  {
-    if (_volumetric_locking_correction)
+    for (_qp = 0; _qp < _qrule->n_points(); ++_qp)
     {
       // Finalize volumetric locking correction
       _Fhat[_qp] *= std::cbrt(ave_Fhat.det() / _Fhat[_qp].det());
       _deformation_gradient[_qp] *= std::cbrt(ave_dfgrd_det / _deformation_gradient[_qp].det());
     }
-
-    computeQpStrain();
   }
+
+  computeQpStrain();
 }
 
 void
 ComputeFiniteStrain::computeQpStrain()
```

Comment:
The `xxxQpxxx` functions usually act on the current QP and not loop over QPs. This is breaking that pattern.

Reply:
Why is this change made only in the finite strain classes and not in the incremental finite strain classes?

Reply:
This does act on the current QP.  ???

Reply:
Sorry, my mistake.  I see what you mean.